packages/react/src/components/CodeSnippet/CodeSnippet.js

```diff
@@ -147,6 +162,23 @@ function CodeSnippet({
     );
   }
 
+  let containerStyle = {};
+  if (type === 'multi') {
+    if (expandedCode) {
+      if (maxExpandedNumberOfRows > 0) {
+        containerStyle.style = {
+          maxHeight: maxExpandedNumberOfRows * rowHeightInPixels,
+        };
+      }
+    } else {
+      if (maxClosedNumberOfRows > 0) {
```

Comment:
```suggestion
    } else  if (maxClosedNumberOfRows > 0) {
```

this does not need to be nested right?

Reply:
It does not, for this PR.
The code is a set up for the next PR [8056](https://github.com/carbon-design-system/carbon/issues/8056) dealing with minHeight it will need to be nested.